### PR TITLE
serverless-dotenv-plugin added

### PR DESCRIPTION
**Pull request by zelenins**
Added dotenv plugin.

When you deploy with NODE_ENV set: NODE_ENV=production sls deploy the plugin will look for a file named .env.production.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REGION = eu_west_2
```
Comment (mjovanovic) [inline .env]: isn't it with dashes instead of underscores? please check

**Review by mjovanovic — CHANGES_REQUESTED** [review]
PR title doesn't match with proposed changes

File: .env (hunk 1 above)
Comment (zelenins) [reply .env]: Yes, it is with dashes. Changed

**Comment by mjovanovic** [comment]
> PR title doesn't match with proposed changes

My fault, I was looking at PR with some filter applied. Sorry

**Pull request merged**